views/default/core/settings/account/username.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+if (!elgg_is_admin_logged_in()) {
+	// only admins are allowed to change the username
+	return;
+}
+
+$user = elgg_get_page_owner_entity();
+if (!($user instanceof ElggUser)) {
+	return;
+}
+
+$body = elgg_view_field([
+	'#type' => 'button',
+	'#class' => 'account-username-change',
+	'#data-toggle-slide' => '0',
+	'href' => '#',
+	'rel' => 'toggle',
+	'value' => elgg_echo('user:username:change'),
+	'class' => 'elgg-button-action',
+	'data-toggle-selector' => '.account-username-change',
+]);
+
+$body .= elgg_view_field([
+	'#type' => 'text',
+	'#help' => elgg_echo('user:username:help'),
+	'#class' => 'hidden account-username-change',
+	'#data-toggle-slide' => '0',
```

Comment:
wanted to make changing the username a more 'special'... like the danger zone (https://github.com/Elgg/Elgg/settings)

don't want an accidental change